ts/features/wallet/onboarding/bancomat/screens/search/SearchBankInfoScreen.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { NavigationEvents } from "react-navigation";
+import { SafeAreaView } from "react-native";
+import { Content } from "native-base";
+import { withLightModalContext } from "../../../../../../components/helpers/withLightModalContext";
+import BaseScreenComponent from "../../../../../../components/screens/BaseScreenComponent";
+import { LightModalContextInterface } from "../../../../../../components/ui/LightModal";
+import I18n from "../../../../../../i18n";
+import { GlobalState } from "../../../../../../store/reducers/types";
+import TosBonusComponent from "../../../../../bonus/bonusVacanze/components/TosBonusComponent";
+import {
+  isError,
+  isLoading,
+  isUndefined
+} from "../../../../../bonus/bpd/model/RemoteValue";
+import { abiListSelector, abiSelector } from "../../../store/abi";
+import {
+  navigateToOnboardingBancomatChooseBank,
+  navigateToOnboardingBancomatSearchAvailableUserBancomat
+} from "../../navigation/action";
+import {
+  loadAbi,
+  searchUserPans,
+  walletAddBancomatBack,
+  walletAddBancomatCancel
+} from "../../store/actions";
+import { fetchPagoPaTimeout } from "../../../../../../config";
+import { emptyContextualHelp } from "../../../../../../utils/emptyContextualHelp";
+import FooterWithButtons from "../../../../../../components/ui/FooterWithButtons";
+import {
+  cancelButtonProps,
+  confirmButtonProps
+} from "../../../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import { SearchBankInfoComponent } from "./SearchBankInfoComponent";
+
+type Props = LightModalContextInterface &
+  ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const renderFooterButtons = (onCancel: () => void, onContinue: () => void) => (
+  <FooterWithButtons
+    type={"TwoButtonsInlineThird"}
+    leftButton={cancelButtonProps(onCancel, I18n.t("global.buttons.cancel"))}
+    rightButton={confirmButtonProps(
+      onContinue,
+      I18n.t("global.buttons.continue")
+    )}
+  />
+);
+
+/**
+ * This screen allows the user to choose a specific bank to search for their Bancomat.
+ * @constructor
+ */
+const SearchBankInfoScreen: React.FunctionComponent<Props> = (props: Props) => {
```

Comment:
Like the route, what you think about rename this screen and update the comment, in order to explain what appens inside this screen?
Maybe we could rename this `BancomatSearchStartScreen` or something similar.